tests/js/handler/handler-caseholder.js

```diff
@@ -0,0 +1,71 @@
+/* Tests for the "express" and "router" module */
+"use strict";
+var fluid        = fluid || require("infusion");
```

Comment:
This and the indentation of the initial requires are fixed throughout the codebase.